_sass/custom/custom.scss

```
@@ -309,3 +316,7 @@ button,
   width: 0%;
   z-index: 10000;
 }
+// Fix: sidebar expander arrow should be cursor pointer (Issue #672)
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)